### DISCUSSION TITLE
Can't add a tooltip to a SVG <path/> element

### DISCUSSION THREAD

**Discussion by TheCodeKat** [primefaces/community#768] [PrimeNG]
Hello!
I've been using PrimeNG components for a while, and I love them. But today I encountered a situation at work where my component's template consists of an SVG with *many* `<path>` elements in it, and I have to attatch a tooltip to each of those paths.

I tried to simply attatch a tooltip to each path like I normally would:
```
<path pTooltip="Hey!">
</path>
```
...but when I hover over the path, the tooltip doesn't seem to appear.

It wasn't until I created a [Stackblitz](https://stackblitz.com/edit/github-jnmek3?file=src%2Fapp.component.ts) that I realized that the tooltip *is* actually appearing...but at the very bottom of the entire SVG element. Also, if you inspect the page, you can see that the inserted tooltip element has its `left` and `top` style properties both set to `-999px`.

So I suppose that I would just need to somehow fix those `left` and `top` values...but I have no idea how.
I tried using the tooltip's `positionTop` property to fix this, but when I used it, the tooltip just didn't appear at all anymore.

I would highly appreciate some help with this, as I need to solve it for my work 🙏🏻
Here's my Stackblitz link: [https://stackblitz.com/edit/github-jnmek3?file=src%2Fapp.component.ts](https://stackblitz.com/edit/github-jnmek3?file=src%2Fapp.component.ts)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please set `[appendTo]="myPath"` and enlarge the viewbox of the path if it's possible. `[appendTo]`="myPath" will append tooltip as a child element of "myPath", but you'll need to enlarge the viewbox to see it. If you check it on devtools, it'll be visible as child of the path element.

**Reply by TheCodeKat** [reply]
Hi,
Thank you for your reply. I ended up making my own tooltip and appending it to the SVG paths using D3.js, but I will try out this solution when I get the time, and I'll report back :)

**Reply by cetincakiroglu** [reply]
Hi,

Could you please share your solution with stackblitz example or repo to fork and dive in? It would help to understand the use case and provide additional aspects to improve the tooltip. Labeled the topic as needs more information to check later.

**Reply by TheCodeKat** [reply]
Sure thing! Just give me a bit of time and I'll share it.

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity
